content/git/git.md

```diff
@@ -1,16 +1,14 @@
 ---
-# This file is used to populate the hub page for the language whose folder it's in. Be sure to create a new version if you create a folder for a new language!
-
-Title: 'Git' # This is the name that appears on the hub page for this language. Pay attention to capitalization and punctuation!
+Title: 'Git'
 Description: 'Git is a popular version control tool that developers use to collaborate, share, and save all relevant versions of their work to avoid it being lost.'
```

Comment:
```suggestion
Description: 'Git is a distributed version control system that developers use to collaborate, share, and save all relevant versions of their work to avoid it being lost.'
```